src/components/pages/About/__tests__/About.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, cleanup } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import About from '../About';
+
+describe('About', () => {
+  afterEach(cleanup);
+
+  test('render About', () => {
+    render(
+      <Router>
+        <About />
+      </Router>,
+    );
+  });
+});
```

Comment:
nit: we don't need to wrap these pages in `Router` only NavBar since it uses `Link`
```suggestion
import { render, cleanup } from '@testing-library/react';
import About from '../About';

describe('About', () => {
  afterEach(cleanup);

  test('render About', () => {
    render(<About />);
  });
});
```